packages/webpack/src/utils/module-federation/dependencies.ts

```diff
@@ -57,12 +62,32 @@ function getLibraryImportPath(
   library: string,
   projectGraph: ProjectGraph
 ): string | undefined {
+  let buildLibsFromSource = true;
+  if (process.env.NX_BUILD_LIBS_FROM_SOURCE) {
```

Comment:
We can find this info from `NX_TASK_TARGET_PROJECT` and `NX_TASK_TARGET_TARGET`? Do we need the extra env var from executor?

Users won't be able to override with CLI options like `--buildLibsFromSource=false` but I think that's fine.

Reply:
I used a separate one because of cases where a new process isn't spawned, those env vars match the parent process (the host) and not the child (the remote).

Reply:
Hmm, interesting. Maybe something to clean-up later if we can move `buildLibsFromSource` to a normal webpack plugin then we wouldn't need all the env vars to be passed like this.